src/ui/public/chrome/api/ui_settings.js

```diff
@@ -0,0 +1,15 @@
+import { metadata } from 'ui/metadata';
+import { Notifier } from 'ui/notify';
+import { UiSettingsClient } from '../../../ui_settings/public/ui_settings_client';
```

Comment:
Is there a specific reason why we wouldn't use `ui/ui_settings/ui_settings_client` instead here?

Reply:
the `ui/` prefix resolves to `ui/public`, which means we might be able to use `ui/../ui_settings/public/ui_settings_client` but that doesn't seem better to me.